front-end/src/components/analysis-page/IntentMenu.jsx

```diff
@@ -100,8 +159,14 @@ class IntentMenu extends Component {
                         aria-label={intent.question}
                         data-testid="intent-menu-list-item"
                     >
-                        <div /* Format the text representation of the intent in the intent menu */>
-                            {this.formatSearchResult(intent.question, searchSlices)}
+                        <div>
+                            {this.addStar(intent)}
+                            {this.addFlag(intent)}
```

Comment:
Maybe make these stack vertically instead of horizontally? Not sure how it would look necessarily, but 2 dots pushes a lot of the content off the screen IMO.
![image](https://user-images.githubusercontent.com/76976842/204161486-ca5cadd4-a3fa-4f55-9754-ca7057bfe76b.png)
Kinda like the above but better spaced.

Also maybe we can make these circles disappear if we have their filter on (since every item would then have them implicitly)

Reply:
+1 on the verticality, I would debate on having the circles disappear with the filter on, simply because small brains like me sometimes need that visual reminder next to the questions, but maybe that's just me